agent/docker-compose.yml

```diff
@@ -12,10 +12,10 @@ services:
       - PMM_DEBUG=1
       - PERCONA_TEST_CHECKS_INTERVAL=10s
       # for local development
-      - PERCONA_TEST_CHECKS_FILE=/srv/checks/custom-checks.yml
+      - PMM_ADVISORS_CHECKS_FILE=/srv/checks/custom-checks.yml
```

Comment:
```suggestion
      - PMM_ADVISOR_CHECKS_FILE=/srv/checks/custom-checks.yml
```
I believe the first noun should be singular in most cases, no?

Reply:
Yeah, I guess we can establish singular as a convention, updated it now.